### DISCUSSION TITLE
issue with starting postal

### DISCUSSION THREAD

**Discussion by lenny1150** [postalserver/postal#2491] [Help with using Postal]
this is what i am getting when i start postal 👍 

root@69-49-228-184:~# postal start
[+] Running 5/5
 ⠿ Container postal-smtp-1      Started                                    1.5s
 ⠿ Container postal-worker-1    Started                                    1.0s
 ⠿ Container postal-cron-1      Started                                    1.2s
 ⠿ Container postal-requeuer-1  Started                                    1.4s
 ⠿ Container postal-web-1       Started                                    1.4s
root@69-49-228-184:~# postal status
NAME                IMAGE                               COMMAND                  SERVICE             CREATED             STATUS              PORTS
postal-cron-1       ghcr.io/postalserver/postal:2.1.4   "/docker-entrypoint.…"   cron                14 seconds ago      Up 12 seconds
postal-requeuer-1   ghcr.io/postalserver/postal:2.1.4   "/docker-entrypoint.…"   requeuer            14 seconds ago      Up 12 seconds
postal-smtp-1       ghcr.io/postalserver/postal:2.1.4   "/docker-entrypoint.…"   smtp                13 seconds ago      Up 12 seconds
postal-web-1        ghcr.io/postalserver/postal:2.1.4   "/docker-entrypoint.…"   web                 13 seconds ago      Up 12 seconds
postal-worker-1     ghcr.io/postalserver/postal:2.1.4   "/docker-entrypoint.…"   worker              13 seconds ago      Up 12 seconds

then when i try to open up 
http://mail.gocopynpaste.com:5000 is not working  

error message  : 

This site can’t be reachedmail.gocopynpaste.com took too long to respond.
Try:

Checking the connection
[Checking the proxy and the firewall](chrome-error://chromewebdata/#buttons)
[Running Windows Network Diagnostics](javascript:diagnoseErrors())
ERR_CONNECTION_TIMED_OUT
 

why did i do wrong in the installation???

please help !!! 

lenny

**Comment by sinnbeck** [comment]
Did you install caddy? https://docs.postalserver.io/install/installation#caddy
I recommend doing that to use the web site

**Comment by willpower232** [comment]
You should not be accessing port 5000 directly, you should use caddy as @sinnbeck mentioned (or nginx or even apache)

**Reply by lenny1150** [reply]
I DID AND IT NOT WORKING  -  
WHAT AM I DOING WRONG I FOLLOWED THESE STEPS FROM THIS WEBSITE 

https://inguide.in/build-postal-smtp-server-send-bulk-emails-ubuntu-18-04/#comment-3327

PLEASE HELP

**Reply by sinnbeck** [reply]
I think you accidentally turned on caps lock :)

Is your server ip 162.241.124.87?

**Reply by lenny1150** [reply]
yes thats my vps ip

**Reply by sinnbeck** [reply]
Ok good. Can you post the output of `docker ps` and `hostname -I`

**Reply by willpower232** [reply]
Those instructions seem incorrect as they suggest you install caddy but then don't use it :thinking: 

It looks like you may need to open some firewall ports if everything is still running.

**Reply by sinnbeck** [reply]
@willpower232 Good points. The guide is also like 2 years old. I assume things might have changed in that time :)

**Reply by lenny1150** [reply]
@willpower232 so which intructions should i use to install postal? and how can i use caddy. And is there a way to open ports ??

**Reply by sinnbeck** [reply]
I just used these: https://docs.postalserver.io/install/installation/

**Reply by lenny1150** [reply]
@sinnbeck yes but it doesnt help with this type of issues - i just send an email to admin of contabo to see if my port 25 is open or not
